### PR TITLE
Add distinctive Cover Image icon

### LINKED ISSUE #5516: Image And Cover Image Icons are Identical
## Issue Overview

The icons for adding an image block and cover image block are the same. This is particularly confusing when you're adding a new block and want to use the quick access menu to the right (see screenshot).

## Version Numbers
Wordpress 4.9.4 (on DreamHost)
Gutenberg plugin 1.3.1
Theme "Advanced Twenty Seventeen"

## Steps to Reproduce

1. Create a new image block, upload an image.
2. Create a new cover image block, upload an image.
NOTE: I'm not entirely sure that uploading is necessary, it might be enough to simply create a block, but that's how I did it.
3. Press enter in a text block to get a new text block.
4. Look to the right of the text area in the new block, note that there are two identical icons. By hovering over them you can see what the each do, but you can't tell from the icon.

## Expected Behavior

I would expect the icons to be different. I was actually rather confused as to why the default image block behavior was graying out my image and having white text over it. Turns out that's the default -cover- image behavior.

## Possible Solution

I'd expect the solution to be changing the icon for the cover image since that's probably the less commonly used option and the icon in use now looks like a standard "image" icon to me.

## Screenshots / Video

image
image

### PR DESCRIPTION
**Pull request by jasmussen**
This fixes #5516, but is blocked from merge by https://github.com/WordPress/gutenberg/pull/5496, as it uses a newer sprite than is merged into Dashicons.

Screenshot:

screen shot 2018-03-13 at 09 49 28

Once #5496 is in a place where we can merge it in, this one will probably require a rebase.

### CODE REVIEW

**Comment by jasmussen** [comment]
Also closes #5575.

File: components/dashicon/index.js [hunk 1]
```diff
@@ -343,6 +351,9 @@ export default class Dashicon extends wp.element.Component {
 			case 'editor-ltr':
 				path = 'M5.52 2h7.43c.55 0 1 .45 1 1s-.45 1-1 1h-1v13c0 .55-.45 1-1 1s-1-.45-1-1V5c0-.55-.45-1-1-1s-1 .45-1 1v12c0 .55-.45 1-1 1s-1-.45-1-1v-5.96h-.43C3.02 11.04 1 9.02 1 6.52S3.02 2 5.52 2zM14 14l5-4-5-4v8z';
 				break;
+			case 'editor-ol-rtl':
```
Comment (aduth) [inline components/dashicon/index.js]: I assume this and "instagram" are expected additions?

**Review by aduth — APPROVED** [review]
👍

File: components/dashicon/index.js (hunk 1 above)
Comment (jasmussen) [reply components/dashicon/index.js]: Yes, they were added to the upstream Dashicons sprite set as part of separate efforts.

**Pull request merged**